Adding information for October 14th International RSE day events (#508)
USRSE/usrse.github.io: _events/2021/2021-10-intnl-rse-day.md

---
title: October 2021 International RSE Day Virtual Events
expires: 2021-10-14
event_date: "October 14, 2021"
layout: event
repeated: false
---

> On October 14th between 11:00 AM and 4:30 PM Eastern time we will be hosting a series of online

Reviewer: ```suggestion
On October 14th between 11:00 AM and 4:45 PM Eastern time we will be hosting a series of online
```

talks and panels with discussion to celebrate the First International RSE Day. Events 

> begin on the hour between 11AM and 4PM Eastern, feel free to join us for some or all

Reviewer: ```suggestion
begin on the hour between 11:00 AM and 4:00 PM Eastern; feel free to join us for some or all
```

> of the days events.

Reviewer 2: "day's" (seem to have lost the suggestion button)

Reviewer: thanks - fixed now

**Background**

> _First International RSE Day Virtual Workshop Events Oct 14, 11AM Eastern - 4:30PM Eastern._

Reviewer: ```suggestion
_First International RSE Day Virtual Workshop Events Oct 14, 11:00 AM Eastern - 4:30 PM Eastern._
```

> _First International RSE Day Virtual Workshop Events Oct 14, 11:00 AM Eastern - 4:30 PM Eastern._

Reviewer: ```suggestion
_First International RSE Day Virtual Workshop Events Oct 14, 11:00 AM Eastern - 4:45 PM Eastern._
```

An NSF-funded workshop titled “Building the research innovation workforce: a workshop

> to identify new insights and directions to advance the research computing community” recently found that collaboration between research software engineers, cyberinfrastructure professionals, and researchers were key to the research computing community (https://www.rcac.purdue.edu/files/ciworkforce2020/report.pdf). 

Reviewer: ```suggestion
to identify new insights and directions to advance the research computing community” recently found that collaboration between research software engineers, cyberinfrastructure professionals, and researchers was key to the research computing community. 
```

> to identify new insights and directions to advance the research computing community” recently found that collaboration between research software engineers, cyberinfrastructure professionals, and researchers were key to the research computing community [https://www.rcac.purdue.edu/files/ciworkforce2020/report.pdf](https://www.rcac.purdue.edu/files/ciworkforce2020/report.pdf). 

Reviewer: ```suggestion
to identify new insights and directions to advance the research computing community” recently found that collaborations between research software engineers, cyberinfrastructure professionals, and researchers were key to the research computing community [https://www.rcac.purdue.edu/files/ciworkforce2020/report.pdf](https://www.rcac.purdue.edu/files/ciworkforce2020/report.pdf). 
```

But what do successful RSE, CI professional, and researcher teams and stakeholder interactions look like? How are they funded and how did people get started in this career?

Join us for a panel discussion and talks to introduce teams of researchers, RSEs, and CI professionals and learn about their experiences in the research computing community.

**Schedule**

October 14th US-RSE Schedule
Start: 8 Pacific / 9 Mountain / 10 Central / 11 AM EST

> | Time (EST)  | Event |  Event Description                         |

Reviewer: ```suggestion
| Time (EDT)  | Event |  Event Description                         |
```

| ----        | ------|------------------------------------------- |

> | 11AM - 11:10AM | Introduction, Blake Joyce/Ian Cosden   | Brief welcome.  |

Reviewer: ```suggestion
| 11:00 AM - 11:10 AM | Introduction, Blake Joyce/Ian Cosden   | Brief welcome.  |
```

| 11:10AM - 12PM | Kickoff Panel     | Speakers introduce themselves and their RSE journey. |
| 12:00 - 12:20 | Nicole Brewer, Purdue     | Title TBD                      |

> | 12:20 - 12:45 | Q&A               |                                |

Reviewer: ```suggestion
| 12:20 PM - 12:45 PM | Q&A               |                                |
```

| 13:00 - 13:20 | Rubern Lara and Mehmet Belgin, Georgia Tech     | Title TBD                     |
| 13:20 - 13:45 | Q&A               |                                |
| 14:00 - 14:20 | Ian Cosden, Princeton     | Building the Princeton Research Software Engineering Group                      |

> | 14:20 - 14:45 | Q&A               |                                |

Reviewer: ```suggestion
| 2:20 PM - 2:45 PM | Q&A               |                                |
```

| 15:00 - 15:20 | Reed Milewicz, Sandia      | Building and Sustaining an RSE Department at Sandia National Laboratory                      |
| 15:20 - 15:45 | Q&A               |                                |

> | 16:00 - 16:20 | TBD, TACC     | Title TBD                      |

Reviewer: ```suggestion
| 4:00 PM - 4:20 PM | TBD, TACC     | Title TBD                      |
```

> | 16:20 - 16:45 | Q&A               |                                |

Reviewer: ```suggestion
| 4:20 PM - 4:45 PM | Q&A               |                                |
```
